assets/section-main-product.css

```diff
@@ -45,6 +46,10 @@
     padding: 0 8rem 0 0;
   }
 
+  .product--right .product__media-list {
```

Comment:
Not related to this PR but I noticed while testing we don't add the same shadow padding for right media that we do for left. Maybe there was a reason behind this, but otherwise maybe we can tackle this in a related follow up as well.

<img src="https://screenshot.click/08-19-moik7-kaaj4.png" width="300" />

Reply:
I think this is actually tackled in Eugene's PR about color scheme as I noticed the same thing 😅

Reply:
Yeah, as Ludo said, it should be fixed with my Color Scheme PR.